marketplace/register.md

```diff
@@ -0,0 +1,5 @@
+curl -X POST \
+   -H "Authorization:Bearer pqOk-ddeyeDNnrSSVwcbuNRMnITwyQcLmE_kbrvV30Y.vUGfxcwdaae3pwORqDED6NNAb2k24AM3tPNm19qS8zw" \
```

Comment:
⚠️ This looks like a secret?

Reply:
This is already expired, so I think it is fine to put it as it is.

Reply:
Ok. But what is the purpose of this file? Just an example? Is it documented somewhere?

Reply:
I keep this file to easily copy and paste for register the app, it is documented in the marketplace app registration documentation but there are some different and typo there so I can not use the command there directly.